src/components/Email/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC, ReactNode } from 'react';
+
+interface EmailPropType {
+  children?: ReactNode;
+}
+
+export const Email: FC<EmailPropType> = (props) => {
```

Comment:
`React.FC` already contains `children` isn't it? Anyways, I don't prefer using `React.FC` as now it's advice not to use it. Let's use the `EmailPropType` directly? 

```
const Email = ({ children }: EmailProps): JSX.Element => { /** */}
```

See https://stackoverflow.com/a/59988810/1190184

Reply:
For some reason, the latest react version doesn't have default `children` for `FC`

Reply:
Do you have any references?

Reply:
https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-typescript-definitions

@agentmilindu